cmd/meroxa/root/functions/logs.go

```diff
@@ -0,0 +1,80 @@
+package functions
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/meroxa/cli/cmd/meroxa/builder"
+	"github.com/meroxa/cli/log"
+	"github.com/meroxa/meroxa-go/pkg/meroxa"
+)
+
+var (
+	_ builder.CommandWithDocs    = (*Logs)(nil)
+	_ builder.CommandWithArgs    = (*Logs)(nil)
+	_ builder.CommandWithClient  = (*Logs)(nil)
+	_ builder.CommandWithLogger  = (*Logs)(nil)
+	_ builder.CommandWithExecute = (*Logs)(nil)
```

Comment:
Did you want to make this command hidden? 
```suggestion
	_ builder.CommandWithExecute = (*Logs)(nil)
	_ builder.CommandWithHidden  = (*Repair)(nil)

```

Reply:
If it's not supposed to be hidden, then can you run `make build`? It's supposed to also create documentations for each command

Reply:
The root command is already marked as hidden: https://github.com/meroxa/cli/blob/5590e3e3efedb8ecf1bd7ed27b510a3f14fed964/cmd/meroxa/root/functions/functions.go#L17. So `make build` won't build any docs for its subcommands